doc/references/ImplementationOfVHDL.rst

```diff
@@ -142,21 +136,77 @@ You can either use a default clock like this:
 
 .. code-block:: VHDL
 
-    -- psl default clock is rising_edge (CLK);
-    -- psl assert always
-    --   a -> eventually! b;
+    default clock is rising_edge (CLK);
+    assert always
+      a -> eventually! b;
 
 or use a clocked expression (note the use of parentheses):
 
 .. code-block:: VHDL
 
-    -- psl assert (always a -> next[3](b)) @rising_edge (clk);
+    assert (always a -> next[3](b)) @rising_edge(clk);
 
 
 Of course only the simple subset of PSL is allowed.
 
 Currently the built-in functions are not implemented.
```

Comment:
I think it'd be good to add a reference to #662.

https://github.com/ghdl/ghdl/blob/master/doc/conf.py#L343

Reply:
Honestly, I'm confused about the various ways of creating a (external) link. When do I use which one?

Reply:
I am confused too. I try to be consistent, but I find it difficult to remember the differences between the multiple alternatives. In this case, `extlinks` is just a shortcut (https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html). Hence, the following are equivalent:

```rst
:issue:`662`
`issue #662 <https://github.com/ghdl/ghdl/issues/662>`_
```

Anyway, I think that `extlinks` are not used at all, even though they are defined. In fact, I don't know if we have any other reference to any issue in the docs. Therefore, it's ok as you did.